### PR TITLE
Add line crossing demo module

### PR DESCRIPTION
**Pull request by abramov-oleg**

### CODE REVIEW

File: requirements/samples.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-scipy~=1.10
\ No newline at end of file
+scipy~=1.10
```
Comment (bwsw) [inline requirements/samples.txt:1]: @abramov-oleg 
Please change the `-samples` dependency to `compose build` with building a local image based on the `-base`. We are going to get rid of samples in the future.

Comment (abramov-oleg) [reply requirements/samples.txt]: Removed dependency on `samples` image.

File: samples/README.md [hunk 2]
```diff
@@ -39,3 +39,11 @@ Preview:
 
 ![](opencv_cuda_bg_remover_mog2-800.webp)
 
+### Line crossing demo
+
+Sample Location: [line_crossing](./line_crossing)
+
+Preview:
+
+![](line-crossing-loop-400.webp.webp)
```
Comment (denisvmedyantsev) [inline samples/README.md:48]: Is it the correct link?

Comment (abramov-oleg) [reply samples/README.md]: Fixed.

File: samples/line_crossing/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+Run the demo:
+
+```bash
+git clone https://github.com/insight-platform/Savant.git
+cd Savant/samples/line_crossing
+git lfs pull
+
+# if you want to share with us where are you from
+# run the following command, it is completely optional
+curl --silent -O -- https://hello.savant.video/opencv_cuda_bg_remover_mog2.html
```
Comment (denisvmedyantsev) [inline samples/line_crossing/README.md:33]:
```suggestion
curl --silent -O -- https://hello.savant.video/cars.html
```

**Review by denisvmedyantsev — APPROVED** [review]

File: samples/line_crossing/README.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+- Always-ON RTSP sink adapter;
+
+**Note**: Ubuntu 22.04 runtime configuration [guide](../../docs/runtime-configuration.md) helps to configure the runtime to run Savant pipelines.
+
+Run the demo:
+
+```bash
+git clone https://github.com/insight-platform/Savant.git
+cd Savant/samples/line_crossing
+git lfs pull
```
[the remark this answers is not in the document]
Comment (abramov-oleg) [reply samples/line_crossing/README.md:29]: Yes, the demo uses an always on RTSP sink, which requires a stub img stored in git LFS.

**Pull request merged**